Library/Homebrew/os/linux/global.rb

```diff
@@ -4,7 +4,7 @@
 # Enables experimental `patchelf.rb` write support.
 HOMEBREW_PATCHELF_RB_WRITE = (
   ENV["HOMEBREW_NO_PATCHELF_RB_WRITE"].blank? &&
-  (ENV["HOMEBREW_PATCHELF_RB_WRITE"].present? || ENV["CI"].blank?)
+  (ENV["HOMEBREW_PATCHELF_RB_WRITE"].present? || ENV["CI"].present? || ENV["HOMEBREW_DEV_CMD_RUN"].present?)
```

Comment:
I think this could be limited just to `HOMEBREW_NO_PATCHELF_RB_WRITE.blank?` if this is moved to test-bot.

Reply:
I'd prefer to roll it out to CI and HOMEBREW_DEV_CMD_RUN first, and give it one stable release to burn in, before rolling it out to everyone.